src/commonMain/kotlin/baaahs/ShowRunner.kt

```diff
@@ -26,7 +27,10 @@ class ShowRunner(
     private var shadersLocked = true
     private var gadgetsLocked = true
 
-    fun getBeatProvider(): Pinky.BeatProvider = beatProvider
+    val currentBeat: Float
```

Comment:
Maybe worth a javadoc here explaining it's continuous from `[0..3)` (or did we decide `[1..4)`)?

Also I'm realizing now that if you access it multiple times within a frame render, the value will change, which is maybe a little weird but probably no big.

Reply:
Done! Also yeah true, but probably not a big deal